packages/server/src/api/controllers/table/index.ts

```diff
@@ -118,6 +119,8 @@ export async function save(ctx: UserCtx<SaveTableRequest, SaveTableResponse>) {
   ctx.eventEmitter &&
     ctx.eventEmitter.emitTable(`table:save`, appId, { ...savedTable })
   ctx.body = savedTable
+
+  savedTable = await processTable(savedTable)
```

Comment:
Why is this needed?

Reply:
Let me see if I can remember, I know there was a reason I discussed it with @mike12345567. Here's the start of a long discussion about it: https://discord.com/channels/733030666647765003/1022445411098710016/1273586235893612636

tl;dr `processTable` wasn't async, I've had to make it async in this PR. But `processTable` was used in a non-async function in `packages/server/src/websockets/builder.ts`. To avoid having to make all of that code async, I instead lifted `processTable` out of that file and put some checks in there instead.